code/modules/shuttle/emergency.dm

```diff
@@ -175,8 +180,26 @@
 	if(ENGINES_STARTED || (!IS_DOCKED))
 		return .
 
+
+	// monkestation addition
+	if(obj_flags & EMAGGED && (!emaglast || emagcooldown + emaglast < world.time) && prob(50))
+		var/cut_time = 2 / rand(1, 5)
```

Comment:
As it stands, this leaves the options of: 1(unchanged), 0(unchanged), 0.66(very slight), 0.5(slight) and 0.4(slight)
I'd suggest simply making a pick proc instead of this random setup.

Reply:
how would i go about doing that?